.github/workflows/canary.yml

```
@@ -0,0 +1,48 @@
+name: canary
+on:
+  - pull_request
+  - push
+jobs:
+  canary:
+    name: testing ${{matrix.package}} with ${{matrix.node}} on ${{matrix.os}}
+    runs-on: ${{matrix.os}}
+    steps:
+      - name: checkout micromark
+        uses: actions/checkout@v2
+      - name: setup node
+        uses: dcodeIO/setup-node-nvm@master
+        with:
+          node-version: ${{matrix.node}}
+      - name: update npm
+        run: npm install -g npm
+      - name: install micromark
+        run: npm install
+      - name: build micromark
+        run: npm run build
```

```suggestion
      - name: micromark
        run: npm install -g npm && npm install && npm run build
```

This should likely work and be uninteresting. And if it doesn’t work, the other CIs should fail

---

Can also use `|`s for multilines, like the next item